drivers/sensor/bosch/bmi08x/Kconfig

```diff
@@ -19,6 +21,7 @@ if BMI08X
 
 choice BMI08X_ACCEL_TRIGGER_MODE
 	prompt "Accelerometer trigger mode"
+	default BMI08X_ACCEL_TRIGGER_NONE
 	default BMI08X_ACCEL_TRIGGER_GLOBAL_THREAD
```

Comment:
Remove the duplicate default line

Reply:
Scrubbed. Also noted the Gyro Kconfig also had this same duplicate, so addressed it as well.